strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-nuget-layout-provider/src/main/java/org/carlspring/strongbox/nuget/file/AssemblyTargetFrameworkAdapter.java

```diff
@@ -0,0 +1,73 @@
+package org.carlspring.strongbox.nuget.file;
```

Comment:
@gokhankuyucak I think we should better name the package `org.carlspring.strongbox.nuget.metadata`

Reply:
+1: I agree.